lib/actions/form.js

```diff
@@ -21,6 +13,16 @@ import {
 
 import { routingQuery } from './api'
 
+const {
+  getDefaultQuery,
+  getTripOptionsFromQuery,
+  getUrlParams,
+  planParamsToQuery
+} = coreUtils.query
+const { getItem, randId } = coreUtils.storage
+const { OTP_API_TIME_FORMAT } = coreUtils.time
+const { isMobile } = coreUtils.ui
```

Comment:
This whole block seems a lot better suited to destructuring in the import statement.

Reply:
Inlining the ones not from query for now.